packages/protocol/contracts/signal/SignalService.sol

```diff
@@ -77,57 +79,78 @@ contract SignalService is ISignalService, EssentialContract {
     }
 
     /// @inheritdoc ISignalService
-    function isSignalReceived(
+    function proveSignalReceived(
```

Comment:
@cyberhorsey @KorbinianK please also review this change as it will affect how proofs are generated and encoded.

Reply:
thanks for the heads up. yes I will have to make some relayer abi changes. I'll make a PR targeting this branch in the morning.